### DISCUSSION TITLE
Kindle Serial Number

### DISCUSSION THREAD

**Discussion by Rich654** [apprenticeharper/DeDRM_tools#2062] [Q&A]
Why do I need to provide the kindle device serial number to the DRM Removal Plugin with Calibri? I don't recall this step being required in the past for DeDRM to work.  I recently installed Calibri on a new computer and imported my library.  I just reinstalled DeDRM plugin to import book from my Kindle and it did not work until I followed the steps in the link below that included adding my Kindle Serial Number. It seemed to me that steps was not needed a few years back when I first used DeDRM (but I could be wrong) !
https://www.cloudwards.net/remove-drm-from-kindle-books/

**Comment by ElleKayEm** [comment]
It's always been needed for books from/for E-ink Kindles.  If you're importing a book from Kindle for PC/Mac, then the key is automatically retrieved.
